### PR TITLE
Add ussd support

### PR DESCRIPTION
**Pull request by fritzbrand**
## Purpose
Add USSD support to ContentRepo

### CODE REVIEW

File: home/models.py [hunk 1]
```diff
@@ -300,6 +300,36 @@ def clean(self, value):
...
+        icon = "user"
+        form_classname = "whatsapp-message-block struct-block"
+
+    def clean(self, value):
+        result = super().clean(value)
+        errors = {}
+
+        if errors:
+            raise StructBlockValidationError(errors)
+        return result
```
Comment (rudigiesler) [inline home/models.py:330]: I think we can probably do without this `clean` method? It doesn't seem to do anything from what I can see

File: home/models.py [hunk 2]
```diff
@@ -300,6 +300,36 @@ def clean(self, value):
         return result
 
 
+class SMSBlock(blocks.StructBlock):
+    message = blocks.TextBlock(
+        help_text="each message cannot exceed 160 characters.",
+        validators=(MaxLengthValidator(160),),
+    )
```
Comment (rudigiesler) [inline home/models.py:307]: I think keep it this way for now, but one thing to think of is that we can have multi-part smses, where the message gets split and sent (and charged) as multiple messages, but then the phone reassembles them on the other side and displays it as a single message. Previously I think we've not used this to keep cost low, but just to keep it in mind. I think we can create another block type to handle that though

File: home/tests/test_content_import_export.py [hunk 3]
```diff
@@ -990,8 +1029,11 @@ def test_web_content(self, impexp: ImportExport) -> None:
         )
 
         orig = impexp.get_page_json()
+        print(orig)
         impexp.export_reimport()
         imported = impexp.get_page_json()
+        print("|||")
+        print(imported)
```
Comment (rudigiesler) [inline home/tests/test_content_import_export.py:1036]: I don't think we need these print statements anymore

**Review by rudigiesler — COMMENTED** [review]
Looks good! The only place I think we would be missing any sort of tests is with the CSV import/export. I might have missed it since CSV diffs are difficult to read, but it seems like we've added the columns but they're all empty, we probably want to add SMS in at least one of the tests to ensure it gets imported and exported correctly

File: home/models.py (hunk 2 above)
Comment (fritzbrand) [reply home/models.py:307]: Cool, understood and agreed

File: home/models.py (hunk 1 above)
Comment (fritzbrand) [reply home/models.py:330]: Yup, thx

File: home/tests/test_content_import_export.py (hunk 3 above)
Comment (fritzbrand) [reply home/tests/test_content_import_export.py:1036]: Weird, I don't see them on my end, will try figure out whats up

**Review by rudigiesler — APPROVED** [review]

**Pull request merged**